undistort_example.py

```diff
@@ -0,0 +1,139 @@
+import open3d as o3d
+import argparse
+import numpy as np
+import mrob
+import math
+
+from pathlib import Path
+from scipy.spatial.transform import Rotation as R
+
+
+def get_intermediate_transform(initial_pose, target_pose, coef):
+    dxi = mrob.geometry.SE3(target_pose @ np.linalg.inv(initial_pose)).Ln()
+    return mrob.geometry.SE3(coef * dxi).T() @ initial_pose
+
+def cut_segment(pcd, theta_min, theta_max, coef):
```

Comment:
![image](https://github.com/p0l0satik/pointcloud_undistort/assets/75482510/dcb905a1-85c0-4777-a65a-3bb66f7c01f3)

Reply:
Also let's use type hints in function signatures